lib/datadog/ci/configuration/settings.rb

```diff
@@ -34,12 +36,11 @@ def self.add_settings!(base)
               # TODO: Deprecate in the next major version, as `instrument` better describes this method's purpose
               alias_method :use, :instrument
 
-              option :trace_flush do |o|
-                o.default { nil }
-              end
+              option :trace_flush
```

Comment:
When nothing is configured like this, is the type nillable?

Reply:
When the there's no type, the validation is disabled completely (https://github.com/DataDog/dd-trace-rb/blob/master/lib/datadog/core/configuration/option.rb#L237) so it's a bit different from nillable (because nillable is a modifier after you select a specific type)